website_backend_views/README.rst

```diff
@@ -0,0 +1,40 @@
+Backend views for website
+=========================
+
+This module allows to embed old style (backend) views into the website frontend. This way, complex forms don't need to be redeveloped but still there's no break in user experience. Ceveat is that this does not work for anonymous users.
+
+This is specifically convenient if you need to use on_change functions, user input dependent domains or got used to be able to create linked objects inline.
+
+Usage
+=====
+
+Install this module with demo data enabled and visit `/website_backend_views/demo`. Then read `view/demo.xml`.
+
+Known issues / Roadmap
+======================
+
+* more testing needed
+* not all css styles from the backend apply (should be just a matter ob having the right html structure/classes attached to the container
+* maybe we can do something for anonymous users?
```

Comment:
Yes, but that's a user. The whole backend framework needs a uid, but I think we could create one without permissions (~portal) and su all calls to that. But it's more tricky than it sounds I think